R-Interface/jasprcpp.cpp

```diff
@@ -259,23 +258,31 @@ void STDCALL jaspRCPP_setDecimalSettings(int numDecimals, bool fixedDecimals, bo
 	rInside[".exactPValues"]		= exactPValues;
 }
 
+void STDCALL jaspRCPP_setFontAndPlotSettings(const char * resultFont, const int ppi, const char* imageBackground)
+{
+	RInside &rInside			= rinside->instance();
+
+	rInside[".resultFont"]			= resultFont;
+	rInside[".imageBackground"]		= imageBackground;
+	rInside[".ppi"]					= ppi;
+
+	jaspRCPP_parseEvalQNT("jaspBase::registerFonts()");
```

Comment:
```suggestion
	jaspRCPP_parseEvalQNT("jaspBase:::registerFonts()");
```
so we wouldn't need to export it in jaspBase?

Also, I'm not 100% sure about which part is responsibile for setting the fonts now. 

It looks like sometimes, jaspBase calls `registerFonts` and sometimes the engine does. Shouldn't it be the case that jaspBase calls this whenever needed (runAnalysis, saveImage, and reWriteImages) and that jaspEngine makes sure .resultFont is available/ updated?

Reply:
> Also, I'm not 100% sure about which part is responsibile for setting the fonts now.
> It looks like sometimes, jaspBase calls registerFonts and sometimes the engine does. Shouldn't it be the case that jaspBase calls this whenever needed (runAnalysis, saveImage, and reWriteImages) and that jaspEngine makes sure .resultFont is available/ updated?

I think the main thing registerFonts seems to do is set `.resultFont` in `graphOptions` in jspGraphs.

As far as it being called in the right places:

Well right now `initEnvironment` calls it, which I assume we'd also run in syntaxmode or so. And the user cant easily switch fonts after that. 

But inside JASP it gets called every time the resultsFont *could have* changed.